_includes/master/kubernetes/k8s_binary_init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I think given that both script operate on the same set of binaries I would:

-  Consolidate into a single script with action parm e.g. `calico-k8s-binaries [download] [install]` or similar
-  Display some sensible help text if neither of those actions are specified.